releases/2023.1/2023.1.md

```diff
@@ -0,0 +1,172 @@
+
+### 4. Migrate `@rest` decorated routes from `werkzeug/flask` to `uvicorn/starlette`
+
+Kytos-ng core has replaced `werkzeug/flag` with `uvicorn/starlette` that way enhancing and solving the following problems:
+
+- `werkzeug 2.1.X` is no longer maintaining support to shutdown `werkzeug` server, which was pivotal for Kytos-ng to ensure a graceful shutdown since it's being embedded in the same process.
+- Although `flask-socketio` server default threading mode is very reliable and works well out of the box for many use cases, it's vulnerable to threads scalability issues.
+- A relatively high-rate of requests, e.g., 500 requests/sec, over 1 minute could lead to overall runtime instability.
+
+`uvicorn/starlette` solves all of these problems while also being optimally designed to be used with `asyncio` and providing a higher capacity to serve requests. If you're upgrading to `2023.1`, the `@rest` decorator is now supposed to take a `request: kytos.core.rest_api.Request` return a `kytos.core.rest_api.JSONResponse` instance, all of the NApps from `kytos-ng` endpoints have been refactored accordingly for `2023.1`, their API specs are still the same.
+
+If you're using a NApp that's not from the `kytos-ng` org and it has `@rest` decorated routes, it needs to be refactored accordingly to be compatible with version `2023.1`. In terms of performance and overall technical requirements, you can find [more information about it on this Kytos-ng GitHub discussion](https://github.com/kytos-ng/kytos/discussions/337). `uvicorn/starlette` also unlocks native Python websockets, which in future versions can expose new types of bidirectional transport. `socket.io` dependency has been removed.
+
+### 5. Stabilize `sdntrace_cp` and general improvements
+
+`sdntrace_cp` is now officially considered stable. All of its API routes are versioned as v1, including supporting OpenAPI validations. Also, `sdntrace_cp` has augmented new states when performing a trace to facilitate understanding an outcome of a trace. In addition, `sdntrace_cp` has removed support to schedule traces to simplify code maintainability.
+
+### 6. General UI fixes and enhancements 
+
+On version `2023.1`, the following UI components were fixed:
+
+- `ui` no logger supports `socket.io`
+- `ui` topology map now displays `metadata.node_name` as a default value when it's defined
+- `mef_eline` now supports `primary_constraints` and `secondary_constraints`, including correctly clearing these components
+
+<!---
+TODO ask Rogerio if he wants to include anything else or screenshots
```

Comment:
cc'ing @rmotitsuki, let me know if you want to also include screenshots or anything else, as we've done in past versions:

![20230614_160511](https://github.com/kytos-ng/kytos-ng.github.io/assets/1010796/d1821347-c22d-4319-886f-4fe947a943e4)

Reply:
I went ahead and included a screenshot